src/libpmemfile/preload.h

```diff
@@ -39,6 +39,7 @@
 
 struct PMEMfilepool;
 struct PMEMfile;
+extern volatile int pause_at_start;
```

Comment:
It made more sense without volatile - it was supposed to prevent compiler from optimizing it out.
Fixed.